easybuild/easyconfigs/w/WRF/WRF-4.6.1-foss-2024a-dmpar.eb

```diff
@@ -0,0 +1,45 @@
+name = 'WRF'
+version = '4.6.1'
+buildtype = 'dmpar'
+versionsuffix = '-%s' % buildtype
+
+homepage = 'https://www.mmm.ucar.edu/models/wrf'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
+
+github_account = 'wrf-model'
+source_urls = [GITHUB_RELEASE]
+sources = ['v%(version)s.tar.gz']
+
+# WRF-4.5.1_netCDF-Fortran_separate_path.patch no longer required for version 4.5.2
+
+checksums = [
+    {'v4.6.1.tar.gz': 'b8ec11b240a3cf1274b2bd609700191c6ec84628e4c991d3ab562ce9dc50b5f2'},
+]
+
+# csh is used by WRF install scripts
+builddependencies = [
+    ('Autotools', '20231222'),
+    ('tcsh', '6.24.13'),
+    ('time', '1.9'),
+    ('Perl', '5.38.2'),
+]
+
+dependencies = [
+    ('JasPer', '4.2.4'),
+    ('netCDF', '4.9.2'),
+    ('netCDF-Fortran', '4.6.1'),
+]
+
+preconfigopts = "export NETCDF=$EBROOTNETCDFMINFORTRAN && export NETCDF_C=$EBROOTNETCDF &&"
```

Comment:
Not a blocker for this PR, but why don't we take care of this in the custom easyblock we already have for WRF, so it's "do once and forget about it", and keeping the easyconfig cleaner?

Even better, it could be done in the `set_netcdf_env_vars` function which is provided by the custom easyblock for netCDF, which is used in both the WPS and WRF easyblocks?

We can go ahead and get this merged and follow up in a subsequent PR on that though...